Assets/Scripts/Elements/Trampoline.cs

```diff
@@ -7,7 +7,7 @@ public class Trampoline : MonoBehaviour {
 	void OnTriggerEnter2D(Collider2D other) {
 		Hero hero = other.GetComponentInParent<Hero> ();
 
-		if (hero != null) {
+		if (hero != null && hero.transform.position.y > transform.position.y) {
```

Comment:
@StefanWerW  Por conta do collider novo, o de interação, quando os heróis encostavam no trampolim ativavam o trigger e eles acabavam pulando e também quando o trampolim caia em cima deles, pois o HeadCollider é um trigger. Então eu forcei o trampolim a ativar somente se o heroi estivesse acima do trampolim.